Cargo.lock

```diff
@@ -367,18 +367,17 @@ dependencies = [
 
 [[package]]
 name = "battery"
-version = "0.7.5"
+version = "0.7.6"
```

Comment:
We should limit dependency updates to PRs that focus solely on updates. Would you be able to revert the changes here?

Reply:
Whoops, missed that I'd changed that.